appserver/admin/gf_template/pom.xml

```diff
@@ -65,6 +65,15 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
The plugin execution is configured in plugin management, but each execution is configured explicitly in a module where it is required.